### DISCUSSION TITLE
Вёрстка. Темы для циклов статей

### DISCUSSION THREAD

**Discussion by solarrust** [doka-guide/content#497] [Идеи]
Привет!

Есть план привлечь сторонних экспертов для написания статей (циклов статей) с углублением в какую-то определённую тему. 

Накидайте темы, в которых вам было бы интересно детально разобраться! Если есть ещё и автор на примете — вообще шикарно =)

Предварительный список:
1. Доступность (Татьяна Фокина, Глафира Жур)
2. SVG (Юля Бухвалова)
3. Вёрстка писем (?)
4. Анимации и трансформации (Юля Миоцен)
5. Перформанс и оптимизация (Иван Акулов)
6. Подключение и оптимизация шрифтов (?)
7. Основы БЭМа (Владимир Гриненко)
8. Сервис-воркеры и офлайн (Максим Сальников)
9. Цифровая гигиена (Андрей Гурылёв)
10. Как делать классные тени и всякие визуальные эффекты (Роман Комаров?)
11. Подготовка графики и видео, оптимизация, обзор форматов (Полина Гуртовая)

**Comment by furtivite** [comment]
Accelerated Mobile Pages (AMP) — мало где рассматривается.

**Reply by pepelsbey** [reply]
Звезда AMP закатилась, плюс это не часть веб-платформы, а технология Google.

**Reply by furtivite** [reply]
@pepelsbey может расширить всё же? Рассказать про турбо-страницы Яндекса, например? Не обязательно в качестве основной программы, может как факультатив? Это может пригодиться начинающим разработчикам, которым могут поставить задачу запилить это.

**Reply by pepelsbey** [reply]
Я бы даже для узких задач писал статьи, которые точно пригодятся всем и не потеряют релевантность через два года, когда какой-нибудь вендор полностью поменяет API или просто закроет проект. Думаю, это хороший повод для обсуждения между лидами: какие технологии мы считаем достойными для документирования.

**Comment by bespoyasov** [comment]
Подготовка графики и видео, оптимизация, обзор форматов, подключение через `srcset`, `<picture>`?

**Reply by bespoyasov** [reply]
Хотя, это мы можем своими силами, наверное 🙂

**Reply by bespoyasov** [reply]
Веб-компоненты бы ещё как-то затронуть, может?

**Reply by pepelsbey** [reply]
Тут не то, чтобы темы, на которые мы сами не сможем написать, просто у нас сейчас задача идти не глубже, а шире. А такие статьи можно поручать внешним, разовым авторам.

**Reply by solarrust** [reply]
>  подключение через `srcset`, `<picture>`

Эти статьи уже в редактуре =) 

Остальные идеи 🔥

**Comment by pepelsbey** [comment]
Точно нужны про доступность и SVG :) Вдобавок:

- Подключение и оптимизация шрифтов
- Основы БЭМа
- Сервис-воркеры и офлайн

**Comment by nlopin** [comment]
* Работа с графикой
* Как делать классные тени и всякие визуальные эффекты

**Comment by solarrust** [comment]
Дополнила список вашими идеями, спасибо огромное! Возвращайтесь, если появятся ещё темы =)

**Comment by furtivite** [comment]
Bootstrap, Tailwind CSS (может ещё другие). Обзорная статья, о том, когда их использовать и зачем?

**Reply by akellbl4** [reply]
Я бы написал про Tailwind, люблю его использовать :) Ну и про бутстрап тоже надо значит

**Comment by Windrushfarer** [comment]
Рисование на CSS может быть? На последнем "Я ❤ фронтенд" Юлия Миоцен как раз воркшоп проводила. Можно ее и позвать :)

**Reply by solarrust** [reply]
Да, Юля уже у нас на карандаше для темы про анимации и трансформации =)
